scripts/utils/integration-tests

```diff
@@ -39,7 +47,14 @@ TESTS_FOLDER="tests"
 
 PLUGINS_PATH=${PLUGINS_PATH:-''}
 
-
+while getopts 's:hnr:' flag; do
```

Comment:
@juliangieseke I would still like giving it a try. Internet suggests `getopt` is buggy but `getopts` should be solid. More over it will be used only on local machines, CI runs with defaults

Reply:
IIRC `getopts` on BSD had some issues, but they might be resolved by now 🤔 also its less flexible as `getopt`. But it should be good enough here 👍